.github/workflows/build-and-test.yml

```diff
@@ -2,7 +2,7 @@ name: Build and Test
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, netstandard-2-0-target ]
```

Comment:
Is this branch gonna be permanent?

Reply:
no, it won't be needed. All this PR is to have one single pipeline to build, test and publish for two platforms, .NET and netstandard2.0.